src/action.ts

```diff
@@ -131,7 +133,9 @@ async function linkAllFilesInMetafile(
 		const downloadDirResult = await getClient().getDownloadDir(searchee);
 		if (downloadDirResult.isErr()) {
 			return downloadDirResult.mapErr((e) =>
-				e === "NOT_FOUND" ? "TORRENT_NOT_FOUND" : e,
+				e === "NOT_FOUND" || e === "UNKNOWN_ERROR"
```

Comment:
not entirely sure about this, but i was getting some failures from it that stemmed from /announces from autobrr - and i traced it to the missing UNKNOWN ERROR thrown from deluge (torrent property missing when it tries to get the download dir)

Reply:
Given that @ShanaryS also said this happens with qbit (fails on downloadDir i think) - i think this might be appropriate. @mmgoodnow

Reply:
I think this would otherwise mean that we are removing the entire premise of UNKNOWN_ERROR existing _OUTSIDE_ of getDownloadDir()? which could ultimately cause some confusion in troubleshooting things in the future...

I'm not sure what those circumstances might be, but I think the distinction might not serve us being ambiguous.

There's probably a middle ground that we could achieve here, but whether or not we return NOT_FOUND and log it elsewhere or use it this way, I don't have any strong opinions on as long as we handle it appropriately in regards to cross-seed's behavior, which it previously was causing qbit and deluge both issues. 

Who knows if it affected or affected other clients? I didn't personally look...